src/govuk/components/modal-dialogue/_modal-dialogue.scss

```diff
@@ -0,0 +1,161 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+@include govuk-exports("govuk/component/modal-dialogue") {
+  $govuk-dialogue-width: 640px;
+
+  .govuk-modal-dialogue,
+  .govuk-modal-dialogue__backdrop {
+    position: fixed;
+    z-index: 0;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+
+  // Hide dialogue when closed
+  .govuk-modal-dialogue {
+    display: none;
+  }
+
+  // Show dialogue when opened
+  .govuk-modal-dialogue--open {
+    display: block;
+  }
+
+  // Wrapper to handle overflow scrolling
+  .govuk-modal-dialogue__wrapper {
+    box-sizing: border-box;
+    display: flex;
+    height: 100%;
+    @include govuk-responsive-padding(7, "top");
+    @include govuk-responsive-padding(7, "bottom");
+    overflow-y: auto;
+    align-items: flex-start; // sass-lint:disable no-duplicate-properties
+    align-items: safe center;
+  }
+
+  // HTML5 dialogue component
+  .govuk-modal-dialogue__box {
+    box-sizing: border-box;
+    display: block;
+    position: relative;
+    z-index: 1;
+    width: 90%;
+    margin: auto;
+    padding: 0;
+    overflow-y: auto;
```

Comment:
This might be okay as it is but just for reference, I notice that the GOV.UK Publishing example currently sets these to:
```
overflow-x: hidden;
overflow-y: scroll;
```
I don't think we currently have an example with page content in the main area so we need to add one in order to check that the main page content is not scrollable and the modal is.